components/Navbar/NavbarComponentsBuilder.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { NavbarItemBuilder } from "./NavbarItem";
+import { NavbarDropdownBuilder } from "./NavbarDropdownBuilder";
+import { ImageComponent } from "../Shared/ImageComponent";
+import { getAssetPath } from "@/lib/getAssetInfo";
+import { NavbarComponents } from "./NavbarComponentsComponent";
+
+type NavbarComponentsBuilderProps = Prisma.NavbarGetPayload<{}>;
+type NavbarComponentBuilderProps = Prisma.NavbarComponentsGetPayload<{}>;
+
+export async function NavbarComponentsBuilder({
+  dateUpdated,
+  logo,
+  id,
+  userUpdated,
+}: NavbarComponentsBuilderProps) {
+  const navbarComponents = await prisma.navbarComponents.findMany({
+    orderBy: {
+      sort: "asc",
+    },
+  });
+  const navbarBuiltComponents = navbarComponents.map(NavbarComponentBuilder);
+  let navbarInfo: NavbarComponentsBuilderProps = {
+    dateUpdated,
+    id,
+    logo,
+    userUpdated,
+  };
+  if (logo)
+    navbarInfo = {
+      dateUpdated,
+      logo: await getAssetPath(logo),
+      id,
+      userUpdated,
+    };
+
+  return (
+    <NavbarComponents componentsSlot={navbarBuiltComponents} {...navbarInfo} />
+  );
```

Comment:
Instead of this, unpack the `navbar` from the previous comment and set the `logo` prop separately.